bin/compile.sh

```diff
@@ -133,9 +133,9 @@ fi
 	# Set any user specified build environment variables.                            #
 	# (Note, this is currently required in order to use Kitura with SwiftNIO.)
 	# ----------------------------------------------------------------------------- #
-	if [ -f $BUILD_DIR/.build_env ]; then
-	  status ".build_env found."
-	  for ENV_VAR in $(cat $BUILD_DIR/.build_env); do
+	if [ -f $BUILD_DIR/.swift-build-env-linux ]; then
+	  status ".swift-build-env-linux found."
+	  for ENV_VAR in $(cat $BUILD_DIR/.swift-build-env-linux); do
```

Comment:
What is the rationale for renaming this file?  One reason we specifically left it as generic as possible is that the env is not Linux-specific (although there is currently no mechanism for making it apply during a Mac build).

We also would like it to be applied during Docker builds, so that the outcome of a Cloud Foundry / Docker build is consistent, and so I don't think it should follow any particular CF naming conventions.

Lastly it would be preferable to keep the name short to avoid mistyping, as users will have to create this file themselves.

Reply:
Further to my earlier comment, I understand that the naming change is to be consistent with the existing `.swift-build-options-linux` facility.   So this does make sense.  I retract my objection 😄